frontend/build-configs/nuernberg/index.ts

```diff
@@ -4,7 +4,7 @@ import publisherText from "./publisherText"
 
 const ANDROID_APPLICATION_ID = "app.entitlementcard.nuernberg"
 const IOS_BUNDLE_IDENTIFIER = "app.sozialpass.nuernberg"
-const ACTIVATION_PATH = "/activation"
+const ACTIVATION_PATH = "activation"
```

Comment:
as this is shared by all build configs it should not be redefined for every build config.

Reply:
moved it to constants folder